<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmallWindow</class>
 <widget class="QWidget" name="SmallWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="actionButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设置为故障</string>
   </property>
  </widget>
  <widget class="QLabel" name="reservationInfoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeSlotComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
